src/main/java/core/basesyntax/serviceimpl/ShopServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.operation.OperationHandler;
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
+    private OperationStrategy operationStrategy;
```

Comment:
The field 'operationStrategy' could be final since it is only assigned in the constructor and should not be changed afterwards.